dev-libs/marisa/marisa-0.2.6.recipe

```diff
@@ -0,0 +1,95 @@
+SUMMARY="Matching Algorithm with Recursively Implemented StorAge"
+DESCRIPTION="Matching Algorithm with Recursively Implemented StorAge (MARISA) is a \
+static and space-efficient trie data structure. And libmarisa is a C++ \
+library to provide an implementation of MARISA. Also, the package of \
+libmarisa contains a set of command line tools for building and \
+operating a MARISA-based dictionary."
+HOMEPAGE="https://github.com/s-yata/marisa-trie"
+COPYRIGHT="2016-2022 Susumu Yata"
+LICENSE="GNU LGPL v2"
+REVISION="1"
+SOURCE_URI="https://github.com/s-yata/marisa-trie/archive/v${portVersion}/marisa-tries-$portVersion.tar.gz"
+CHECKSUM_SHA256="1063a27c789e75afa2ee6f1716cc6a5486631dcfcb7f4d56d6485d2462e566de"
+SOURCE_DIR="marisa-trie-$portVersion"
+
+ARCHITECTURES="all !x86_gcc2"
+SECONDARY_ARCHITECTURES="x86"
+
+libVersion="0.0.0"
+libVersionCompat="$libVersion compat >= ${libVersion%%.*}"
+
+PROVIDES="
+	marisa$secondaryArchSuffix = $portVersion
+	lib:libmarisa$secondaryArchSuffix = $libVersionCompat
+	"
+REQUIRES="
+	haiku$secondaryArchSuffix
+	"
+
+PROVIDES_devel="
+	marisa${secondaryArchSuffix}_devel = $portVersion
+	devel:libmarisa$secondaryArchSuffix = $libVersionCompat
+	"
+REQUIRES_devel="
+	marisa$secondaryArchSuffix == $portVersion base
+	"
+
+if [ -z "$secondaryArchSuffix" ]; then
```

Comment:
This can't be build for primary architecture so doing it like this won't produce a tools package on 32bit